### ISSUE TITLE
Discuss removing suppression for settersHaveSinceTag

### ISSUE THREAD

**Issue by stoyanK7** [sevntu-checkstyle/sevntu.checkstyle#1022]
In https://github.com/checkstyle/checkstyle/pull/13455 we add a `MatchXPath` check to ensure that all property setters have a `@since` property in their Javadoc. For example
```java
/**
...
@since 1.1
*/
public final void setSomething() ..
```

In https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/1022 we add a suppression for all checks in this repository because the CI in checkstyle repo is failing
https://checkstyle.semaphoreci.com/jobs/b80149c4-f163-4908-a4ab-98a19bcb625e
```
ERROR] [checkstyle] [ERROR] /home/semaphore/checkstyle/.ci-temp/sevntu.checkstyle/sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/annotation/ForbidAnnotationCheck.java:66:5: All property setters of a module should contain a @since tag. [[ERROR] /home/semaphore/checkstyle/.ci-temp/sevntu.checkstyle/sevntu-checks/config/ant-phase-verify.xml:116: Checkstyle failed: Got 115 errors and 0 warnings.] 01:07
[ERROR] [checkstyle] [ERROR] /home/semaphore/checkstyle/.ci-temp/sevntu.checkstyle/sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/annotation/ForbidAnnotationCheck.java:79:5: All property setters of a module should contain a @since tag. [settersHaveSinceTag]
...
[ERROR] /home/semaphore/checkstyle/.ci-temp/sevntu.checkstyle/sevntu-checks/config/ant-phase-verify.xml:116: Checkstyle failed: Got 115 errors and 0 warnings.
```

I am not sure if we want to add those `@since`s in this project or if it's possible. The purpose of this issue is to add an `until ...` on the line above the suppression of this check.

**Comment by nrmancuso** [comment]
I think it is reasonable to add such tags to property setters anywhere really, let's see what others think.

**Comment by rnveach** [comment]
I don't believe this repo has those things documented, so it will be regression hunting to identify all the versions to add. The custom branch I used referenced in the main repo's issue should be able to be used for this.

**Comment by romani** [comment]
This repo don't need this metadata for now, if we have time in future we will do this
